docs/core/tutorials/create-custom-template.md

```diff
@@ -203,6 +203,10 @@ If you created the template on your local file system at *C:/Users/\<USER>/Docum
 dotnet new -u C:/Users/<USER>/Documents/Templates/GarciaSoftware.ConsoleTemplate.CSharp
 ```
 
+> [!NOTE]
+> To uninstall the template from your local file system, you will need to fully-qualify the path. For example, *C:/Users/\<USER>/Documents/Templates/GarciaSoftware.ConsoleTemplate.CSharp* will work, but *./GarciaSoftware.ConsoleTemplate.CSharp* from the containing folder will not.
```

Comment:
will need -> need (avoid future tense when possible)

fully-qualify -> fully qualify (Do not hyphenate -ly adverbs.)

Reply:
I'll adjust those.

That "-ly adverbs" concept was totally new to me for some reason, so thank you! (I'm not 100% in agreement with the [motivation given by the Chicago style](https://www.dailywritingtips.com/adverbs-and-hyphens/), but it certainly seemed well-established, and I'll gladly adapt.)